modules/ossm-validating-smcp.adoc

```diff
@@ -33,10 +34,31 @@ You can verify the Service Mesh control plane installation in the {product-title
 . Navigate to *Operators* -> *Installed Operators*.
 . Select the `<istio-system>` namespace.
 . Select the {SMProductName} Operator.
-. Click the *Istio Service Mesh Control Plane* tab.
-. Click the name of your control plane, for example `basic`.
-. To view the resources created by the deployment, click the *Resources* tab. You can use the filter to narrow your view, for example, to check that all the *Pods* have a status of `running`.
-. If the SMCP status indicates any problems, check the `status:` output in the YAML file for more information.
+.. Click the *Istio Service Mesh Control Plane* tab.
+.. Click the name of your control plane, for example `basic`.
+.. To view the resources created by the deployment, click the *Resources* tab. You can use the filter to narrow your view, for example, to check that all the *Pods* have a status of `running`.
+.. If the SMCP status indicates any problems, check the `status:` output in the YAML file for more information.
+.. Navigate back to *Operators* -> *Installed Operators*.
+
+. Select the OpenShift Elasticsearch Operator.
```

Comment:
In line 27 referred to this as the **Red Hat** Elasticsearch Operator. Does that same terminology apply here?

Reply:
Red Hat is not technically part of the Operator name.